_scripts/deploy.sh

```diff
@@ -5,7 +5,17 @@
 
 cd "$(dirname "$0")" || exit 1
 
-export IMAGE_PREFIX=deisci VERSION=v2-alpha
+
+VER=v2-alpha
+# note that we must check TRAVIS_PULL_REQUEST first because, for pull request builds, TRAVIS_BRANCH will contain the name of the branch that the PR is _targeting_ (not the name of the branch it is trying to merge).
+# see https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables for an explanation of TRAVIS_PULL_REQUEST and TRAVIS_BRANCH
+if [[ $TRAVIS_PULL_REQUEST != "false" ]; then
```

Comment:
Travis CI won't ever call the `deploy` target on a PR build, so this code is effectively unreachable, according to https://docs.travis-ci.com/user/deployment/#Pull-Requests